### PR TITLE
Added VyOS

### PR DESCRIPTION
**Pull request by conceptual-perception**
I read and tried to follow the contributing guidelines.

I added VyOS. I'm active on the Cisco Study Group Discord.

### CODE REVIEW

File: virl-base-images/vyos/readme.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+This works for CML v2.7.2.
+
+The Hypervisor used to create the `.qcow2` is Proxmox.
```
Comment (xorrkaz) [inline virl-base-images/vyos/readme.md:3]: While I am a Proxmox user and like it a lot, Proxmox is not [yet] a supported CML platform, and many users will not have it.  I appreciate the detailed instructions here, but could you also include `virt-install` [instructions](https://docs.vyos.io/en/equuleus/installation/virtual/libvirt.html)?

File: node-definitions/vyos/VyOS-Router.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
+id: VyOS-Router
+general:
+  nature: router
+  description: Opensource Router and Firewall Platform
```
Comment (xorrkaz) [inline node-definitions/vyos/VyOS-Router.yaml:4]: Might be good to use `|-` and include a bit more details such as a link to the VyOS docs.

File: node-definitions/vyos/VyOS-Router.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    has_loopback_zero: true
+    physical:
+      - eth0
+      - eth1
+      - eth2
+      - eth3
+      - eth4
+      - eth5
+    serial_ports: 1
+    min_count: 1
```
[the remark this answers is not in the document]
Comment (conceptual-perception) [reply node-definitions/vyos/VyOS-Router.yaml]: modified on my fork.

File: node-definitions/vyos/VyOS-Router.yaml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    default_count: 6
+    loopback:
+      - lo
+ui:
+  visible: false
+  label_prefix: VyOS-
```
[the remark this answers is not in the document]
Comment (conceptual-perception) [reply node-definitions/vyos/VyOS-Router.yaml]: modified on my fork

File: virl-base-images/vyos/readme.md (hunk 1 above)
Comment (conceptual-perception) [reply virl-base-images/vyos/readme.md]: Added 
https://github.com/conceptual-perception/cml-community/blob/master/virl-base-images/vyos/virt-install-instructions.md

File: node-definitions/vyos/VyOS-Router.yaml [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+      - eth3
+      - eth4
+      - eth5
+    serial_ports: 1
+    min_count: 1
+    default_count: 6
+    loopback:
+      - lo
+ui:
+  visible: false
```
[the remark this answers is not in the document]
Comment (conceptual-perception) [reply node-definitions/vyos/VyOS-Router.yaml]: modified on my fork

**Comment by conceptual-perception** [comment]
I'm still learning git. I nuked what I was doing on accident trying to squash. I'll figure it out and try again.

**Comment by conceptual-perception** [comment]
The above should be resolved. It was very messy -- I had to ask a colleague to help find the disconnected commits.

**Comment by xorrkaz** [comment]
Thanks!  Great work.

**Pull request merged**